src/static/order.php

```diff
@@ -22,100 +67,54 @@
             </div>
             <div class="rightside">
                 <div class="nav-linkwrapper">
-                    <a href="index.html">Home</a>
+                    <a href="index.php">Home</a>
                 </div>
 
                 <div class="nav-linkwrapper">
-                    <a href="orderform.html">Order Form</a>
+                    <a href="order.php">Order Form</a>
                 </div>
                 <!-- 
                 <div class="nav-linkwrapper">
                     <a href="login.html">Login</a>
                 </div>
                 -->
-
             </div>
         </div>
     </div>
     <div class="content-wrapper">
        <div class="order-form-wrapper">
-            <h1>Thank you for your order!</h1>
+            <h1>Online Orders</h1>
+            <p>Use the form below to send us your order.</p>
+            
+            <form action="order.php" method="post">
+                <label for="user_id">User ID:</label><br>
+                <input type="text" id="user_id" name="user_id" value="userid#" required><br><br>
+                <label for="email">Enter your email:</label><br>
+                <input type="email" id="email" name="email" required><br><br>
+                <label for="address">Shipping address:</label><br>
+                <input type="text" id="address" name="address" value="123 shark lane Sharksville CO 12345" required><br><br>
+                <label for="itemid">Item ID:</label><br>
+                <input type="text" id="itemid" name="itemid" value="item id#" ><br>
+                <label for="itemquant"> Item quantity:</label><br>
+                <input type="number" id="itemquant" name="itemquant" min="1" ><br><br>
```

Comment:
we should eventually add back in the maximum and have it set to whatever the current amount is.